lib/booking.rb

```diff
@@ -0,0 +1,33 @@
+require 'date'
+# require 'pry'
+module Hotel
+  class Booking
+    attr_accessor :start_date :end_date :room
+
+    def initialize(start_date, end_date)
+      @start_date = start_date
+      @end_date = end_date
+      @room = room
+
+    end
+
+    def start_date
+      return @start_date.strftime
```

Comment:
This logic should be done in the initialize and then just use `attr_accessor`s to access the updated data